test/complexity_test.cc

```diff
@@ -28,6 +28,8 @@ int AddComplexityTest(std::string test_name, std::string big_o_test_name,
   AddCases(TC_JSONOut, {{"\"name\": \"%bigo_name\",$"},
                         {"\"run_name\": \"%name\",$", MR_Next},
                         {"\"run_type\": \"aggregate\",$", MR_Next},
+                        {"\"repetitions\": %int,$", MR_Next},
+                        {"\"threads\": %int,$", MR_Next},
```

Comment:
Here and elsewhere - `threads` is never(?) auto-detected, thus it is not some `%int`, but should instead be explicitly checked.

Reply:
Alright, done!

Reply:
Thanks.
Same for `repetitions`, i 'forgot' to mention it here.